frontend/src/__tests__/cypress/cypress/utils/oc_commands/distributedWorkloads.ts

```diff
@@ -49,14 +49,21 @@ export const createKueueResources = (
  * @param {string} clusterQueueName - The name of the cluster queue to delete.
  * @param {string} resourceFlavor - The name of the resource flavor to delete.
  * @param {string} projectName - The namespace/project in which the resources exist.
+ * @param {object} options - Configuration options for the deletion operation.
+ * @param {number} options.timeout - Timeout in milliseconds for the command (only used when wait is true).
+ * @param {boolean} options.wait - Whether to wait for the deletion to complete (default: true).
+ * @param {boolean} options.ignoreNotFound - Whether to ignore errors when resources are not found (default: false).
  * @returns {Cypress.Chainable<CommandLineResult>} A Cypress chainable resolving with the result of the deletion command.
  */
 export const deleteKueueResources = (
   localQueueName: string,
   clusterQueueName: string,
   resourceFlavor: string,
   projectName: string,
+  options: { timeout?: number; wait?: boolean; ignoreNotFound?: boolean } = {},
 ): Cypress.Chainable<CommandLineResult> => {
+  const { timeout, wait = true, ignoreNotFound = false } = options;
+
   // Create the OC command to delete the resources
   const ocCommand = `
       oc delete LocalQueue ${localQueueName} -n ${projectName} && 
```

Comment:
Or add it the fix here:
```suggestion
      oc delete LocalQueue ${localQueueName} -n ${projectName} || : ;
```